### ISSUE TITLE
Write a script to scrape flower image from Google Image

### ISSUE THREAD

**Issue by indiejoseph** [ml-hongkong/hongkong_flowers#1]
[repo](https://github.com/ml-hongkong/hongkong_flowers/tree/master/scrapper)

**Comment by indiejoseph** [comment]
Asked Nelson So to help us to update the flowers list, he is organizer of [柴娃娃植物網](https://www.facebook.com/groups/cwwHKplant), which a local flower interest.

**Comment by alt0xFF** [comment]
Ok when this is done, I can rerun the scraper

**Comment by alt0xFF** [comment]
@indiejoseph i think for the list, you will want to keep the hierarchical categories.  I suspect it may be helpful.

**Comment by indiejoseph** [comment]
@alt0xFF Yes, Nelson will give us a list with `name`, `scientific name`,  `family name`
